src/modules/theme/constants/fonts.js

```diff
@@ -2,22 +2,43 @@
 
 export type FontsType = {
   fontSizeAdjust: number,
-  decorativeFontFamily: string,
+  decorativeFontBold: string,
+  decorativeFontRegular: string,
   decorativeFontSize: string,
   decorativeLineHeight: number,
-  contentFontFamily: string,
+  contentFontRegular: string,
+  contentFontBold: string,
+  webviewFontFamilies: string,
   contentFontSize: string,
   contentLineHeight: number,
   standardParagraphMargin: string,
   subTitleFontSize: string
 }
 
-export default {
+export const arabicFonts: FontsType = { // used for 'ar', 'fa' and 'ku'
```

Comment:
I'm thinking about discussing a font switch on the next conference. I think nobody would notice a change actually...
Google Noto is quite cool when it comes to glyph support: https://www.google.com/get/noto/

Reply:
What do you think? Should I put it on the agenda?

Reply:
Should this also replace Raleway? I like Raleway though :D